### PR TITLE
feat: verify source without destination URL

### LINKED ISSUE #170: Support `--verify` flag without destination
Currently, there is no support for user to verify only if it is a valid source or not, it would be good if we can verify the only source

example

`abc import --verify=true  --src_type=postgress --src_uri=http://user:pass@host:port/db`

* abc version latest

### PR DESCRIPTION
**Pull request by harsh-2711**
### Required for all PRs:

- [ ] README.md updated (if needed)
- [x] Passes `gofmt` and `golint`

Fixes: #170

### CODE REVIEW

File: cmd/abc/appbase_import.go [hunk 1]
```diff
@@ -334,3 +339,17 @@ func verifyConnections(adaptors map[string]adaptor.Adaptor) error {
 	}
 	return nil
 }
+
+func verifyConnectionsWithoutDestination(srcConfig map[string]interface{}) error {
+	var config = make(map[string]interface{})
+	for k, v := range srcConfig {
+		config[k] = v
+	}
+	ad, _ := adaptor.GetAdaptor(srcConfig["_name_"].(string), config)
```
Comment (jeet-parekh) [inline cmd/abc/appbase_import.go]: Skipping the error check here would cause a failure, if we don't support the source or the source type is misspelt.

**Review by jeet-parekh — CHANGES_REQUESTED** [review]
Also, have you tested this? Does it work fine?

**Comment by harsh-2711** [comment]
> Also, have you tested this? Does it work fine?

Yes. It does work fine. But I haven't tested importing with a wrong source type. I'll test it and will make changes for the same.

File: cmd/abc/appbase_import.go (hunk 1 above)
Comment (harsh-2711) [reply cmd/abc/appbase_import.go]: Are you suggesting appname and File path check? I have made a commit for it. Please check.

Comment (jeet-parekh) [reply cmd/abc/appbase_import.go]: I meant turning that line `ad, _ := ` into `ad, err := `.
And then adding an `if err != nil` check right below.

`adaptor.GetAdaptor` would return an error if you pass it a wrong name.

Comment (harsh-2711) [reply cmd/abc/appbase_import.go]: Done👍🏻

File: cmd/abc/appbase_import.go [hunk 2]
```diff
@@ -341,11 +341,32 @@ func verifyConnections(adaptors map[string]adaptor.Adaptor) error {
 }
 
 func verifyConnectionsWithoutDestination(srcConfig map[string]interface{}) error {
+
```
Comment (jeet-parekh) [inline cmd/abc/appbase_import.go]: You could remove these custom checks (added in commit 33043c4) now.
It's taken care of by the individual adaptor's Verify methods.

**Review by jeet-parekh — CHANGES_REQUESTED** [review]
Looks good. Remove the custom verification code, and we could merge it then.

File: cmd/abc/appbase_import.go (hunk 2 above)
Comment (harsh-2711) [reply cmd/abc/appbase_import.go]: Done

**Review by jeet-parekh — APPROVED** [review]
LGTM

**Pull request merged**